src/Display.php

```diff
@@ -90,7 +90,30 @@ public function add_body_class( $classes ) {
 	 */
 	protected function template( $part, $args = array() ) {
```

Comment:
Short array syntax please
```suggestion
	protected function template( $part, $args = [] ) {
```

Reply:
Don't worry about this - keep the file consistent and we'll update them all in one go if/when we add coding standards.